jcl/src/openj9.jvm/share/classes/com/ibm/jvm/CRIUSupport.java

```diff
@@ -0,0 +1,45 @@
+/*[INCLUDE-IF CRIU_SUPPORT]*/
+package com.ibm.jvm;
```

Comment:
Please move the package declaration after the copyright notice (like most files).